lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -167,6 +167,7 @@ def call(params)
               fail "Client id cannot be empty" if client_id.blank?
 
               c = all_clients.detect { |specific_client| specific_client.id == client_id }
+              fail "Not found any client with id: #{client_id}" if c.nil?
```

Comment:
We should log warning message and skip this client instead of failing, similar to next check "client out of segment":

params.gdc_logger.warn "Client #{client_id} is not found"

Reply:
Fixed as comment. Thank you